Code/GraphMol/Descriptors/Property.cpp

```diff
@@ -105,7 +105,7 @@ int Properties::registerProperty(PropertyFunctor *prop) {
 std::vector<std::string> Properties::getAvailableProperties() {
   registerDescriptors();
   std::vector<std::string> names;
-  BOOST_FOREACH (boost::shared_ptr<PropertyFunctor> prop,
+  for (boost::shared_ptr<PropertyFunctor> prop :
```

Comment:
so _technically_ this is an extra copy of the shared_ptr, which invokes an extra increment/decrement of the reference count.

But I think that using the C++11 range-based for loop makes pop out, so I'm a LGTM as is.